imagemounter/filesystems.py

```diff
@@ -636,14 +636,18 @@ def mount(self):
         os.environ['LVM_SUPPRESS_FD_WARNINGS'] = '1'
 
         # find free loopback device
-        self._find_loopback()
+        # No need to find loopback device if using nbd device
+        if "nbd" not in self.volume.get_raw_path():
```

Comment:
Is this case also true for other block devices? If the LVM is already mounted on another loopdevice for instance.